cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -106,8 +122,22 @@ func createAzureManagerInternal(configReader io.Reader, discoveryOpts cloudprovi
 		return nil, err
 	}
 
+	retryBackoff := wait.Backoff{
```

Comment:
"added retry for creatingAzureManager in case of throttled requests"

Reply:
This got upstreamed in master: https://github.com/kubernetes/autoscaler/pull/6550.
Thus, this change in this PR will be equivalent to cherry-pick.